package.json

```diff
@@ -111,7 +112,7 @@
       "eslint --cache --fix"
     ]
   },
-  "packageManager": "pnpm@7.3.0",
+  "packageManager": "pnpm@7.4.0-1",
```

Comment:
Perhaps we should revert this at the meantime as `pnpm add -g pnpm` updates to 7.3.0 only, and otherwise this may confuse new contributors.

Reply:
Sorry, this is miss committed.